.config/nextest.toml

```diff
@@ -1,2 +1,6 @@
 [profile.default]
 slow-timeout = { period = "10s", terminate-after = 3 }
+
+[[profile.default.overrides]]
+filter = 'test(state_migration_actor_bundle_runs)'
+slow-timeout = { period = "120s", terminate-after = 3 }
```

Comment:
Why is it so slow? Can we make it faster? I'd very much prefer a regular cargo test to be speedy.

Reply:
Or alternatively, have it run in ci only using profiles: https://nexte.st/book/configuration.html#profiles

Reply:
I'll keep in the regular tests for now. Otherwise, it would be out of sight and never fixed. :)